portfolio/src/main/webapp/style.css

```diff
@@ -52,6 +63,34 @@ ul#navigation li {
    background-color: beige;
 }
 
+.slideshowContainer {
+    position: relative;
+    margin: auto;
+}
+
+.individualSlide {
+    padding-top: 20px;
+    display: none;
+}
+
+.dotContainer {
+    text-align: center;
+}
```

Comment:
nit: follow a consistent style with naming, it looks like existing identifiers use lower-case dashed (`greeting-container`)

(remember to update HTML, CSS, and JS)